examples/tv_to_rdf.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+
+# Converts an tag/value file to RDF format.
+# Usage: tv_to_rdf <tagvaluefile> <rdffile>
+
+if __name__ == '__main__':
```

Comment:
Could you make the code below a function and move the import at the top?
And call that function in this `if __name__ == '__main__'` block instead?